packages/admin/blocks-admin/src/iframebridge/HoverPreviewComponent.tsx

```diff
@@ -38,13 +38,13 @@ export const HoverPreviewComponent = ({ children, componentSlug }: PropsWithChil
     return (
         <sc.Root
             ref={rootEl}
-            onMouseOver={(e) => {
+            onMouseOver={(e: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
Why is this explicit type annotation necessary? Wouldn't it be better to fix the type of `Root`?

Reply:
Fixed in 5487afda5dea233d5f53fbe951888c082da877ce

I don't really understand why but changing `styled("div")` to `styled(Box)` fixes the type. Maybe not an optimal solution but since this has no effect on styling an it's the only place where it's really an issue it's probably fine.